README.md

```diff
@@ -49,32 +72,47 @@ Use the `Makefile` — run `make help` for available commands!
 * Artifactory credentials to access our private
 [Canton Enterprise Docker images](https://digitalasset.jfrog.io/ui/repos/tree/General/canton-enterprise-docker/digitalasset/canton-enterprise/latest)
 
-Docker Compose will automagically build the [image for the HTTP JSON API service and Trigger service](daml-service/)
-from the release JAR files.
-
+Docker Compose will automagically build the [image for the HTTP JSON API service](daml-service/) from the release JAR file.
 Check the [`.env`](./.env) file to know which Canton and SDK versions are being used.
 
-⚠️ You can test different versions changing `CANTON_VERSION` and `SDK_VERSION`, however, there is no guarantee that it will be
-compatible with the currently committed configurations and Grafana dashboards.
```

Comment:
This warning got lost somehow, we need to keep it

Reply:
My bad.  Somehow I lost it.  I put it back.  Thank you for pointing this out.

Reply:
We should just refer to the section below ('Accessing Daml Enterprise Docker Images') as this is described well there.

Reply:
This sentence regarding how to test different versions is actually now in the TLDR section, so it hasn't been lost.